static/app/views/starfish/components/spanDescription.tsx

```diff
@@ -37,17 +37,18 @@ export function DatabaseSpanDescription({
   groupId,
   preliminaryDescription,
 }: Omit<Props, 'op'>) {
-  const {data: indexedSpans, isFetching: areIndexedSpansLoading} = useIndexedSpans({
-    search: MutableSearch.fromQueryObject({'span.group': groupId}),
-    sorts: [INDEXED_SPAN_SORT],
```

Comment:
@gggritso this is a weird one, seems like this sort was never being applied before? Without removing this sort, the test would fail because the test expected no sort in the query object

https://github.com/getsentry/sentry/blob/40ee1fe571af1e543a3216cacc420b4d42372aa1/static/app/views/performance/database/databaseSpanSummaryPage.spec.tsx#L179

Reply:
odd, we can probably add the sort in a separate pr.